how-we-work/mozmeao.md

```diff
@@ -1,25 +1,67 @@
 # How we work in 2018
 
-Our team manages it's work via a [Github project in the MozMEAO organization](https://github.com/orgs/mozmeao/projects/2). The goal of this team-centric Kanban board is to help manage backend and infra/SRE tasks. This will allow us to:
+#### tl;dr
 
-- prioritize and assign work in a fair and balanced way
+Our team manages it's work via two Github projects separated into [milestones](https://github.com/orgs/mozmeao/projects/3) and [tasks](https://github.com/orgs/mozmeao/projects/2). 
+
+The goal of this process is to:
+
+- establish and follow a team roadmap
+- organize team-wide projects ("swarming") to foster collaboration, prevent isolation, spread knowledge, and bring ⚡️energy⚡️ to cloud engineering
+- plan, prioritize and assign work in a fair and balanced way
 - communicate status within the MozMEAO team and marketing organization
 - identify risks, blockers and high priority issues
-- foster collaboration on the server side
 
 All data in the project is "standalone": we try to keep all links and references as public as possible, but there are obviously tasks that include sensitive data. These sensitive tasks are managed internally, and can also tracked in private Bugzilla issues.
```

Comment:
Might as well add the missing "be" ("...and can also _be_ tracked...") while we're making edits. 😀